src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <!-- rev'ed to 4.0.1.0 even though 4.0.0.0 never shipped so that we can drop pre-release down to beta -->
     <ProjectGuid>{4C2655DB-BD9E-4C86-83A6-744ECDDBDF29}</ProjectGuid>
-    <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
+    <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
```

Comment:
I found a couple others that were broken like System.Buffers and fixed them. I also update the other projects that set DocumentationFile to use the same pattern and cleaned up some more redundant AssemblyName Properties.